packages/design-system-web/src/components/geov-authority-lookup-explorer/geov-authority-lookup-explorer.tsx

```diff
@@ -169,6 +181,32 @@ export class GeovAuthorityLookupExplorer {
     }
   }
 
+  getGeovistoryData() {
+    this.title = TITLE_GEOVISTORY;
+    this.uriData = [];
+    const kw = this.keywords.trim().split(' ').map(word => `*${word}*`).join(' AND ');
```

Comment:
my prettier complains. Yours?

Reply:
Done, with ESlint plugin (which not activated)